services/frontend-service/src/ui/components/ReleaseCard/ReleaseCard.scss

```diff
@@ -58,6 +58,7 @@
 }
 
 .release-environment {
+  white-space: nowrap;
   background-color: grey;
   .mdc-evolution-chip__text-label {
     color: white;
```

Comment:
I know this isn't in the PR but would be nice if this (and the next lines about colours) used variables instead
Also would be nice if the font weight and size (in the next 2 lines) used the respective [typography](https://github.com/freiheit-com/kuberpult/blob/3619b6bf89eb4a070449213ac93e1d9433c0c9cd/services/frontend-service/src/assets/_base.scss#L34)

Reply:
I am using ".sub-headline1" now, it seems to be the closest to what we have in the design.